Lib/asyncio/base_events.py

```diff
@@ -647,6 +651,72 @@ def _getaddrinfo_debug(self, host, port, family, type, proto, flags):
         return await self.run_in_executor(
             None, socket.getnameinfo, sockaddr, flags)
 
+    async def sock_sendfile(self, sock, file, offset=0, count=None,
+                            *, fallback=None):
+        if self._debug and sock.gettimeout() != 0:
+            raise ValueError("the socket must be non-blocking")
+        self._check_sendfile_params(sock, file, offset, count)
+        if fallback:
+            return await self._sock_sendfile_fallback(sock, file,
+                                                      offset, count)
+        else:
+            try:
+                return await self._sock_sendfile_native(sock, file,
+                                                        offset, count)
+            except _SendfileNotAvailable as exc:
+                if fallback is None:
+                    return await self._sock_sendfile_fallback(sock, file,
+                                                              offset, count)
+                else:
+                    raise RuntimeError(exc.args[0]) from None
+
+    async def _sock_sendfile_native(self, sock, file, offset, count):
+        raise _SendfileNotAvailable("Fast sendfile is not available")
```

Comment:
i'd change to 'sendfile syscall is not available'

Reply:
or to 'os.sendfile is not available'

'fast sendfile' is rather cryptic and doesn't really tell anything to the user